### PR TITLE
Use a fixed fybrik/optimizer image from ghcr.io repository

### PR DESCRIPTION
**Pull request by shlomitk1**
Signed-off-by: Shlomit Koyfman <shlomitk@il.ibm.com>
Fixes https://github.com/fybrik/fybrik/issues/1531

This PR uses ghcr.io/fybrik/optimizer:0.0.1 image. The version is specific to the solver version and is not related to Fybrik release.

### CODE REVIEW

File: charts/fybrik/values.yaml [hunk 1]
```diff
@@ -215,8 +215,7 @@ manager:
   solver:
     # image of the container with solver binary and libs
     # when specified, the solver will be deployed in the manager pod
-    image: "optimizer"
-    imagePullPolicy: "Always"
+    image: "ghcr.io/fybrik/optimizer:master"
```
Comment (revit13) [inline charts/fybrik/values.yaml:218]: when creating a new release for Fybrik the optimizer tag should use the release tag?

Comment (shlomitk1) [reply charts/fybrik/values.yaml:218]: We are not creating a new image for optimizer for each release. But maybe it should represent the tool and the version underneath.

Comment (revit13) [reply charts/fybrik/values.yaml:218]: It might be better to use a fixed version for the optimizer rather than master for debugging purposes.

Comment (shlomitk1) [reply charts/fybrik/values.yaml]: Ok, using optimizer:0.1.

File: charts/fybrik/values.yaml [hunk 2]
```diff
@@ -215,8 +215,7 @@ manager:
   solver:
     # image of the container with solver binary and libs
     # when specified, the solver will be deployed in the manager pod
-    image: "optimizer"
-    imagePullPolicy: "Always"
+    image: "ghcr.io/fybrik/optimizer:0.1"
```
Comment (revit13) [inline charts/fybrik/values.yaml:218]: i think its better to use a major.minor.patch format for the tag following [semantic versioning](https://semver.org/)

Comment (shlomitk1) [reply charts/fybrik/values.yaml]: Changed to 0.0.1

**Review by revit13 — APPROVED** [review]

**Pull request merged**